pkg/scheduler/framework/interface.go

```diff
@@ -145,6 +145,50 @@ func NewPodsToActivate() *PodsToActivate {
 	return &PodsToActivate{Map: make(map[string]*v1.Pod)}
 }
 
+// StatusError is an implementation of error.
+type StatusError struct {
```

Comment:
Why we need this? Status contains all the message we need. It seems we're adding complexities.

Reply:
Same view here, why're we creating a new struct.

Reply:
True, maybe the tests should just use pointers to errors that can be easily compared, as opposed to `fmt.Errorf` and `errors.New`